bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/read/RowAdapter.java

```diff
@@ -50,6 +50,10 @@ public Result adaptResponse(Row response) {
 
     for (com.google.cloud.bigtable.data.v2.models.RowCell rowCell : response.getCells()) {
 
+      if (!rowCell.getLabels().isEmpty()) {
+        continue;
+      }
+
```

Comment:
I don't understand why we would drop cells that have labels set

Reply:
As per [this comment](https://github.com/googleapis/java-bigtable-hbase/blob/41a3d98cb4561917ac91e1ea9207b99e1f3e0067/bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/read/FlatRowAdapter.java#L51), I assumed that If any cell contains Labels then we treat those as an internal cell.

Also as of now [AbstractBigtableTable#get](https://github.com/googleapis/java-bigtable-hbase/blob/41a3d98cb4561917ac91e1ea9207b99e1f3e0067/bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/AbstractBigtableTable.java#L262-L264) & [AbstractBigtableTable#getScanner](https://github.com/googleapis/java-bigtable-hbase/blob/41a3d98cb4561917ac91e1ea9207b99e1f3e0067/bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/AbstractBigtableTable.java#L290-L295) have adapted with FlatRowAdapter which already has this check but [AbstractBigtableTable#append](https://github.com/googleapis/java-bigtable-hbase/blob/41a3d98cb4561917ac91e1ea9207b99e1f3e0067/bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/AbstractBigtableTable.java#L507-L526) & [AbstractBigtableTable#increment](https://github.com/googleapis/java-bigtable-hbase/blob/41a3d98cb4561917ac91e1ea9207b99e1f3e0067/bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/AbstractBigtableTable.java#L530-L542) have adapted with `models.Row` for which I missed to add this check in RowAdapter.